django_project/frontend/src/components/Footer/index.jsx

```diff
@@ -32,20 +32,41 @@ export default function Footer() {
   // Set width of logo
   // Not working using css on firefox
   $(".page__header-logo").width($(".page__header-link").width());
+
+  // Only show mobile nav on small screens
+  const [activePanel, setActivePanel] = useState("map");
+  const navItems = [
+    { key: "context", icon: "📄", label: "Context" },
+    { key: "indicator", icon: "📊", label: "Indicator" },
+    { key: "filters", icon: "🔍", label: "Filters" },
+    { key: "map", icon: "🗺️", label: "Map" },
+    { key: "widgets", icon: "📈", label: "Widgets" },
+  ];
+
+  // Emit a custom event for panel switching (for integration)
+  const handleNavClick = (key) => {
+    setActivePanel(key);
+    const event = new CustomEvent("mobilePanelSwitch", { detail: { panel: key } });
```

Comment:
Hi @prishashethh
Is the click handler done? As i see it does not do anything
And there is no reference to **CustomEvent**